packages/terra-clinical-data-grid/src/subcomponents/Row.jsx

```diff
@@ -87,6 +94,7 @@ class Row extends React.Component {
         data-row
         data-row-id={rowId}
         data-section-id={sectionId}
+        data-column-allow-highlight={(!isSelected && allowColumnHighlighting) ? true : null}
```

Comment:
Can `data-column-allow-highlight` be renamed to `data-allow-column-highlight` to sound similar to allowColumnHighlighting?

Reply:
I was thinking the other data props reference the object first (e.g. data-row, data-row-foo, data-section-foo, data-column-foo) that it may make more sense to rename `allowColumnHighlighting` to `columnAllowHighlight` to match `data-column-allow-highlight` ?

Reply:
Have updated to be `data-allow-column-highlight`